adoc/deployment-airgapped.adoc

```diff
@@ -387,6 +387,26 @@ This will print out a list of the images skuba is expecting to use on the cluste
 Mirror those and setup the crio-registries to point to the location they are mirrored at.
 ====
 
+[TIP]
+====
+These images need to be available in the external and internal mirrors at the time you try to deploy SUSE CaaS Platform.
+Image tags are varies depending on your version of kubernetes you install.
+:===
+CATEGORY:IMAGE:URL
+KUBEADM:hyperkube:registry.suse.com/caasp/v4/hyperkube
+KUBEADM:pause:registry.suse.com/caasp/v4/pause
+KUBEADM:coredns:registry.suse.com/caasp/v4/coredns
+KUBEADM:etcd:egistry.suse.com/caasp/v4/etcd
+ADDONS:cilium:registry.suse.com/caasp/v4/cilium-init
+ADDONS:cilium:registry.suse.com/caasp/v4/cilium-operator
+ADDONS:cilium:registry.suse.com/caasp/v4/cilium
+ADDONS:dex:registry.suse.com/caasp/v4/caasp-dex
+ADDONS:gangway: registry.suse.com/caasp/v4/gangway
```

Comment:
Are these whitespaces supposed to be here? The other entries do not have them.

Reply:
Thanks for checking mistakes.